scripts/seed-database/write/lowcal_sessions.sql

```diff
@@ -0,0 +1,63 @@
+-- insert lowcal_sessions overwriting conflicts
+CREATE TEMPORARY TABLE sync_lowcal_sessions (
+  id uuid,
+  data jsonb,
+  created_at timestamptz,
+  updated_at timestamptz,
+  email text,
+  flow_id uuid,
+  deleted_at timestamptz,
+  submitted_at timestamptz,
+  has_user_saved boolean,
+  sanitised_at timestamptz,
+  locked_at timestamptz,
+  allow_list_answers jsonb
+);
+
+\copy sync_lowcal_sessions FROM '/tmp/lowcal_sessions.csv' WITH (FORMAT csv, DELIMITER ';');
+
+INSERT INTO
+  lowcal_sessions (
+    id,
+    data,
+    created_at,
+    updated_at,
+    email,
+    flow_id,
+    deleted_at,
+    submitted_at,
+    has_user_saved,
+    sanitised_at,
+    locked_at,
+    allow_list_answers
+  )
+SELECT
+  id,
+  data,
+  created_at,
+  updated_at,
+  email,
+  flow_id,
+  deleted_at,
+  submitted_at,
+  has_user_saved,
+  sanitised_at,
+  locked_at,
+  allow_list_answers
+FROM
```

Comment:
Totally understand the utility of this, but it's one we've avoided in the past so that we don't encourage pulling/keeping user data on our personal machines - I wonder if we could sync just a subset of columns here (notably omitting `email` and full `data`) and still get same local utility? I think something like `SELECT null as email, '{}' as data...` would work and preserve expected table schema?

`alllow_list_answers` is non-personally-identifiable but will still include application type, project types, proposal description and more for any record that has not null `submitted_at` which I'm guessing is what you're most after?